src/psyclone/tests/psyGen_test.py

```diff
@@ -58,10 +58,11 @@
     OMPParallelDoDirective, InlinedKern, \
     OMPParallelDirective, OMPDoDirective, OMPDirective, Directive, \
     ACCEnterDataDirective, ACCKernelsDirective, HaloExchange, Invoke, \
-    DataAccess, Kern, Arguments, CodedKern
+    DataAccess, Kern, Arguments, CodedKern, Argument
```

Comment:
pylint complains that Reference is unused.

Reply:
Deleted